packages/d/dpp/xmake.lua

```diff
@@ -32,7 +36,9 @@ package("dpp")
         os.rmdir("include/dpp/nlohmann")
 
         os.cp(path.join(package:scriptdir(), "port", "xmake.lua"), "xmake.lua")
-        import("package.tools.xmake").install(package)
+        import("package.tools.xmake").install(package, {
+            ccache = false -- little fix because it doesn't pass CI check on github
+        })
```

Comment:
I tested it when enable ccache, it works on ci. https://github.com/xmake-io/xmake-repo/actions/runs/2432134090